### PR TITLE
ignore mousedown when originating from contextmenu click

### PR DESCRIPTION
**Pull request by jakubzitny**

### CODE REVIEW

**Comment by jakubzitny** [comment]
not sure about the `e.stopPropagation()`

File: src/resizable-crop.jsx [hunk 1]
```diff
@@ -97,6 +97,12 @@ class ResizableCrop extends React.Component {
   }
 
   _handleMouseDown(e) {
+    // NOTE: The mousdown event originated in right/contextmenu click. Ignore.
+    if (e.which === 3) {
+      e.stopPropagation()
```
Comment (jankuca) [inline src/resizable-crop.jsx]: Let's remove this and let the user decide how to handle right clicks.

Comment (jakubzitny) [reply src/resizable-crop.jsx]: @jankuca done.

Comment (jakubzitny) [reply src/resizable-crop.jsx]: @jankuca actually, thinking about it one more time, the stopPropagation() should be there. Or just the return should be moved below the stopPropagation. All the methods here are stopping propagation and there is no way for the user to distinguish contextmenu from the event (that's caught inside the library).

Either stopPropagation or pass a callback (or the `e`vent) to let user know about it.

**Pull request merged**